backend/app.ts

```diff
@@ -85,11 +107,49 @@ passport.use(new GoogleStrategy({
         }
     }));
 
-    passport.use(new LocalStrategy({
-                usernameField: 'email',
-                passwordField: 'password'
-        },
-        async (email: string, password: string, done: Function) => {
+passport.use(new FacebookStrategy({
+    clientID: facebookClientID,
+    clientSecret: facebookAppSecret,
+    callbackURL: "http://localhost:8000/auth/facebook/callback",
```

Comment:
no change needed but just keep in mind this URL will have to eventually be abstracted out, since we won't be using locahost after we deploy this code!

Reply:
how do you think we would solve this moving forward, so we can stop hardcoding in the localhost URL?

Reply:
My guess is to use an ENV variable?

Reply:
correct! you'd pass in the API_URL or similar as an env var